### DISCUSSION TITLE
Add to able to choose serialize/deserialize format

### DISCUSSION THREAD

**Discussion by aablinov** [outline/rich-markdown-editor#308] [General]
Hello, what do u think about to add props to choose serialize/deserialize format? I think it's good practice serialize to JSON because it's more friendly to machine processing. 

Something like this:

```
<Editor onChange=((value) => value('json')) serializeFrom={'json'} value={{JSON HERE}}/>
```
and need change in index.tsx function ```createState``` -> ```EditorState.fromJSON``` depending on settings.

**Comment by tommoor** [comment]
Markdown in + out is pretty much what makes this project unique – there are lots of prosemirror powered editors out there including the very pluggable [remirror](https://github.com/remirror/remirror) that enable this level of control… 

Having said that it's not a huge lift and could be useful if it could be implemented quite seamlessly – `serializeFrom` probably not even needed just detect `value` if it's a string or an object?
